tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/instrumented_assembly_generator_helper.h

```diff
@@ -0,0 +1,330 @@
+#pragma once
+#include "../com_ptr.h"
+#include "../cor_profiler.h"
+#include "../log.h"
+#include "../util.h"
+#include "instrumented_assembly_generator_consts.h"
+#include "method_info.h"
+#include "../../../../shared/src/native-src/dd_filesystem.hpp"
+#include <fstream>
+#include <random>
+#include <iomanip>
+#include <utility>
+#if !_WIN32
+#include <unistd.h>
+#endif
+
+using namespace shared;
```

Comment:
not sure to understand why you add this, since you still prefix the symbols from `shared` with `shared::`

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a